libs/designsystem/src/lib/components/checkbox/checkbox.component.html

```diff
@@ -4,7 +4,7 @@
     [checked]="checked"
     [disabled]="disabled"
     (ionChange)="onChecked($event.detail.checked)"
-    [attr.aria-labelledby]="labelText ? _labelId : ''"
+    [attr.aria-labelledby]="text ? _labelId : ''"
```

Comment:
Consider fallback to null to avoid rendering the attribute altogether if no text:
```suggestion
    [attr.aria-labelledby]="text ? _labelId : null"
```